2016/help.html

```diff
@@ -0,0 +1,155 @@
+---
+layout: default
+title: How you help
+slug: help
+permalink: /help/
+---
+
+<div class="section page-topper">
+  <h1>How you help</h1>
+  <p>Whether you’re a funder, a donor, a volunteer or someone who visits our sites, we couldn’t do it without you.</p>
+  <h2>We want to give you a great big, heartfelt
+    <span class="hearts">
+      <i class="fa fa-heart" aria-hidden="true"></i>
+      <i class="fa fa-heart" aria-hidden="true"></i>
+      <i class="fa fa-heart" aria-hidden="true"></i>
+      <i class="fa fa-heart" aria-hidden="true"></i>
+      <i class="fa fa-heart" aria-hidden="true"></i>
+      ‘thank you.’
+    </span>
+  </h2>
+</div>
+
+<div class="section section--white">
+  <h1>Our fantastic funders</h1>
+  <p>We are profoundly grateful to:</p>
+  <ul class="funders">
+    <li><a href="http://www.omidyar.com/">Omidyar Network</a>, <a href="http://www.potterfoundation.com/">The David &amp; Elaine Potter Foundation</a>, and <a href="https://www.opensocietyfoundations.org/">Open Society Foundations</a> — for their support across everything we do.</li>
+    <li><a href="http://www.hewlett.org/">The William and Flora Hewlett Foundation</a>, <a href="https://www.ucl.ac.uk/">University College London</a> and <a href="https://www.microsoft.com/en-gb/about/charities-communities/charity-support/">Microsoft</a> — for funding our research.</li>
+    <li><a href="https://indigotrust.org.uk/">The Indigo Trust</a> — for aiding our work with partners in sub-Saharan Africa.</li>
+    <li>The <a href="https://www.digitalnewsinitiative.com/">Google Digital News Initiative</a> — for funding the launch of Alaveteli Professional.</li>
+    <li><a href="http://www.ned.org/">National Endowment for Democracy</a> — for funding a WriteInPublic project for the Middle East and North Africa region</li>
+    <li><a href="http://www.jrct.org.uk">The Joseph Rowntree Charitable Trust</a> — for supporting our efforts to engage with a younger audience.</li>
+    <li><a href="http://www.makingallvoicescount.org/">Making All Voices Count</a> — for supporting Alaveteli for Liberia.</li>
+    <li><a href="https://www.google.com/">Google Inc.</a> — for sponsoring TICTeC 2017.</li>
+  </ul>
+</div>
+
+<div class="section section--no-padding">
+  <img src="{{ site.baseurl }}/assets/img/hands-on-keyboard.jpg" alt="" style="width: 100%;">
+</div>
+
+<div class="section section--white">
+  <h1>Our loyal donors</h1>
+  <p>Your money goes towards all sorts of things, including:</p>
+  <ul class="help-list">
+    <li>Keeping our servers spinning</li>
+    <li>Responding to our users’ support mail</li>
+    <li>Improving our sites</li>
+    <li>Maintaining our modest offices</li>
+    <li>Helping staff meet up face-to-face when needed</li>
```

Comment:
Let's avoid the repetition of 'help' and change this to "Transporting staff to meetings when they need to meet up face-to-face".